manim/utils/tex_file_writing.py

```diff
@@ -200,14 +196,14 @@ def compile_tex(tex_file: Path, tex_compiler: str, output_format: str) -> Path:
     result = tex_file.with_suffix(output_format)
     tex_dir = config.get_dir("tex_dir")
     if not result.exists():
-        command = tex_compilation_command(
+        args = tex_compilation_command(
             tex_compiler,
             output_format,
             tex_file,
             tex_dir,
         )
-        exit_code = os.system(command)
-        if exit_code != 0:
+        cp = subprocess.run(args, stdout=subprocess.DEVNULL)
```

Comment:
I haven't found the time to test this myself, but is there a reason that you're voiding the output instead of storing it (`capture_text=True`)?

Reply:
This is equivalent to `">", os.devnull` in `tex_compilation_command` before this commit.